packages/forms-web-app/src/controllers/before-you-start/enforcement-notice.js

```diff
@@ -0,0 +1,69 @@
+const logger = require('../../lib/logger');
+const { createOrUpdateAppeal } = require('../../lib/appeals-api-wrapper');
+const { VIEW } = require('../../lib/views');
+const {
+  validEnforcementNoticeOptions,
+} = require('../../validators/before-you-start/enforcement-notice');
+
+exports.getServiceNotAvailableWhenReceivedEnforcementNotice = (req, res) => {
```

Comment:
Isn't this functionality included in the use-a-different-notice ticket?

Reply:
Removed